### PR TITLE
Delta Station: Fix some vendors in science.

### PR DESCRIPTION
**Pull request by DogeDogIs**
Fix some vendors that was not there. Mr. Changs is back on science baby (Yes back, he was there before)

## What Does This PR Do

Fix somes vendors in delta science that was supposed to be there but it isn't. When the science PR was pulled I had some erros and tried to fix in the Byond editor but didn't fixed all, now I have SDMM back.

## Why It's Good For The Game

Just some fixs. Its strange to see a empty space, a refill and coffe seeds on the ground.

## Images of changes

image
image
image
image

## Testing

Made a local host and saw everything with the eyes.

## Changelog
:cl:
fix: Fixed some vendors missing in Delta Science
/:cl:

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]
Looking good to me

File: _maps/map_files/Delta/delta.dmm [hunk 1]
```diff
@@ -154543,7 +154539,7 @@ aaa
 aaa
 abj
 abj
-oEG
+btJ
```
Comment (Contrabang) [inline _maps/map_files/Delta/delta.dmm:154542]: image
image

**Review by Contrabang — APPROVED** [review]
LGTM

**Comment by DogeDogIs** [comment]
bro again
fuck you bot

**Comment by DogeDogIs** [comment]
god damn it

**Comment by DogeDogIs** [comment]
its fixed

**Review by S34NW — APPROVED** [review]

**Pull request merged**